qark/plugins/Dynamically_Loading_Code.py

```diff
@@ -5,25 +5,58 @@
 from modules import common
 from lib.pubsub import pub
 import lib.plyj.parser as plyj
+import lib.plyj.model as m
 
 
 class DynamicallyLoadingCodePlugin(IPlugin):
     DEX_CLASS_LOADER = r'DexClassLoader'
     CLASS_LOADER = r'loadClass'
     DYNAMIC_BROADCAST_RECEIVER = r'registerReceiver'
+    global list_BR
+    list_BR = []
```

Comment:
Are you sure you need a list here? It seems you are adding `file_name` to it. Will the same `file_name` need to be in the list more than once or is it possible to use a `set()`?